class.js

```diff
@@ -0,0 +1,71 @@
+class Human {
+  constructor(name, age, data) {
+    this.name = name;
+    this.age = age;
+    this.dataOfBirth = data;
+  }
+
+  displayInfo() {
+    console.log(`Human: name - ${this.name}, age - ${this.age}, birthday - ${this.dataOfBirth}`);
+  }
+}
+
+class Employee extends Human {
+  constructor(name, age, data, salary, department) {
+    super(name, age, data);
+    this.salary = salary;
+    this.department = department;
+  }
+
+  displayInfo() {
+    super.displayInfo();
+    console.log(`Employee: salary - ${this.salary}, department - ${this.department}`)
+  }
+}
+
+class Developer extends Employee {
+  constructor(name, age, data, salary) {
+    super(name, age, data, salary);
+    this.department = 'Developer';
+  }
+}
+
+class Manager extends Employee {
+  constructor(name, age, data, salary) {
+    super(name, age, data, salary);
+    this.department = 'Manager';
+    this.listDev = [];
+  }
+
+  addDeveloper(name, age, data, salary) {
+    this.listDev.push(new Developer(name, age, data, salary, this.name));
```

Comment:
```suggestion
    this.listDev.push(new Developer(name, age, data, salary));
```
Разработчик принимает только 4 аргумента, а под менеджера там свойства нет:)